pages/faqs.md

```diff
@@ -53,16 +53,56 @@ If your command-line client was installed manually, please check the path of the
 
-### What if my issue is not listed on this page?
-  If your problem hasn't been resolved or reported, then [create an issue](https://github.com/exercism/cli/issues/new) with the following information:
+## Mentored Mode
 
-  1. The output of the `exercism troubleshoot` command.
-  1. Details on the issue you are experiencing.
-  1. Instructions on how to reproduce the issue.
+### My queue hasn't updated in a while - is there something wrong?
+While wait times can be longer than normal for a number of reasons (number of mentors, time of year), it can also appear that the queue is not updating. This is not unusual and you should see movement fairly soon. If you your wait is far longer than the average, refer to [Opening an Issue](#opening-an-issue) below.
 
-## Exercism Team FAQs
+### How many mentors does my track have? I only see a handful on the website.
+The website only shows mentors who have provided bio information for the website, not the actual number of mentors who are actively reviewing solutions. Rest assured, there are mentors working through their queues so hang in there!
 
-### What happened to my Team from the old site?
+### I submitted the wrong solution / file. What can I do?
+There isn't a way to delete submissions so just submit the right file and it will show up as a new iteration in the queue. Feel free to leave a note for your mentor with the original submission clarifying what happened.
+
+### How can I report abuse or examples of bad mentoring?
+Please check our [Code of Conduct](https://exercism.io/code-of-conduct) for more information about our expectations of conduct. If you would like to report something, please reach out to us at [abuse@exercism.io](mailto:abuse@exercism.io?subject=%5BCoC%5D]) and we will try to fix or resolve the issue respecting both you and your privacy.
+
+## Improving Exercism
+
+### This is great! How do I get involved?
+There are a few different ways - becoming a mentor, managing a language track or reporting (or addressing!) issues on GitHub. You can see more on the [How to Contribute page](https://exercism.io/contribute). Version 3 of Exercism is also being built right now and needs help from track maintainers. The [v3 repository](https://github.com/exercism/v3) has more information.
 
+### How do new language tracks get added to the site?
+A new language track gets created when a member of the community takes the lead on it and becomes a maintainer of the track. If you'd like to get involved in helping set one up, there are [instructions here](https://github.com/exercism/request-new-language-track/blob/master/README.md).
+
+### Why are issues taking a while to get resolved?
+Version 3 of Exercism is being actively worked on, which means that contributors and maintainers are splitting their time between the current version (V2) and the next.
+
+### What happened to my Team from the old site?

```

Comment:
This should probably point to `teams.exercism.io` and not be in this section

Reply:
I've moved this to the top and updated the text.